play-2.4/swagger-play2/build.sbt

```diff
@@ -6,6 +6,8 @@ checksums in update := Nil
 scalaVersion:= "2.11.6"
 crossScalaVersions := Seq("2.11.6", "2.11.7")
 
+resolvers += "scalaz-bintray" at "http://dl.bintray.com/scalaz/releases"
```

Comment:
do you need to add this? i though scalaz was in maven central now. see https://github.com/functional-streams-for-scala/fs2/issues/258#issuecomment-124001847

(p.s. you should never add an http resolver since it's a security risk)